csp/benchmarks/stats/basic.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import time
+from datetime import datetime, timedelta
+
+import csp
+
+
+class StatsBenchmarkSuite:
+    def setup(self):
+        self.st = datetime(2020, 1, 1)
+        self.N = 1_000
+        self.ARRAY_SIZE = 100
+        self.TEST_TIMES = [self.st + timedelta(seconds=i) for i in range(self.N)]
+        self.RANDOM_VALUES = [np.random.normal(size=(self.ARRAY_SIZE,)) for i in range(self.N)]  # 100 element np array
+        self.DATA = list(zip(self.TEST_TIMES, self.RANDOM_VALUES))
+        self.INTERVAL = 500
+        self.NUM_SAMPLES = 100
+
+    def time_stats_qtl(self):
+        def g_qtl():
+            data = csp.curve(typ=np.ndarray, data=self.DATA)
+            median = csp.stats.median(data, interval=self.INTERVAL)
+            csp.add_graph_output("final_median", median, tick_count=1)
+
+        qtl_times = []
+
+        for _ in range(self.NUM_SAMPLES):
```

Comment:
Can we use the python interface to [timeit](https://docs.python.org/3/library/timeit.html#python-interface) to do the actual timing to cut down on boilerplate?

Reply:
agreed